File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.java
class Solution {
    public int solution(int a, int b, int n) {
        int answer = 0;
        
        while(n >= a){
            int bottle = b * (n / a); // 받은 병수
            answer += bottle;
            n = n % a + bottle;
        }
        return answer;
    }
}

File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.py
def solution(a, b, n):
    # 2개를 주면 1개를 반환해주는 것 , a를 주면 b병을 준다.
    answer = 0
    total = n
    
    while(total >= a):
        answer += total // a * b
        total = total // a * b + total % a
        print(total)
        
    return answer